docs/source/config.json

```
@@ -42,7 +42,7 @@
       "Federated schema checks": "/managed-federation/federated-schema-checks",
       "Deployment best practices": "/managed-federation/deployment",
       "Opt-in error reporting": "/managed-federation/error-reporting",
-      "Studio features": "https://www.apollographql.com/docs/studio/federated-graphs/"
+      "Studio features": "https://www.apollographql.com/docs/studio/federated-graphs"
```

It looks like https://www.apollographql.com/docs/studio/federated-graphs now redirects to https://www.apollographql.com/docs/graphos/graphs/federated-graphs/ so we might as well send folks directly there :) 
```suggestion
      "Studio features": "https://www.apollographql.com/docs/graphos/graphs/federated-graphs"
```